temba/orgs/models.py

```diff
@@ -86,6 +86,9 @@
 ACCOUNT_SID = 'ACCOUNT_SID'
 ACCOUNT_TOKEN = 'ACCOUNT_TOKEN'
 
+TWILIO_COMPLIANT_API_ACCOUNT_SID = 'TWILIO_COMPLIANT_API_ACCOUNT_SID'
+TWILIO_COMPLIANT_API_ACCOUNT_TOKEN = 'TWILIO_COMPLIANT_API_ACCOUNT_TOKEN'
```

Comment:
You are right about this, but I think it actually points out that maybe we should be using a slightly different approach. Twilio and Nexmo being org scoped is actually a regret of ours in the platform, it causes some issues in various places.

Especially since this channel type has different possible endpoints (while Twilio always goes to the same place) this configuration does belong on the channel objects with its own endpoint as opposed to being org level. (ie, more similar to the Kannel endpoint than the current Twilio endpoint)

What do you think?